Assets/Editor/DumpInputActionReferences.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+internal static class DumpInputActionReferences
+{
+    private static void DumpReferences(StringBuilder sb, string prefix, InputActionReference[] references)
+    {
+        sb.Append(prefix + ":\n");
+        foreach (var reference in references)
+        {
+            var s = reference.action != null ? "Yes" : "No";
+            sb.Append($"- {reference.name} (Resolved: {s}, Asset: {reference.asset})\n");
+        }
+    }
+
+    private static void DumpReferences()
+    {
+        var sb = new StringBuilder();
+        DumpReferences(sb, "Loaded objects", Object.FindObjectsByType<InputActionReference>(
+            FindObjectsInactive.Include, FindObjectsSortMode.InstanceID));
+        DumpReferences(sb, "All objects:", Resources.FindObjectsOfTypeAll<InputActionReference>());
+        Debug.Log(sb.ToString());
+    }
+
+    [UnityEditor.MenuItem("QA Tools/Dump Input Action References to Console", false, 100)]
+    private static void Dump()
+    {
+        DumpReferences();
+    }
+}
```

Comment:
Should this be part of our Input Debugger interface instead?

Reply:
Maybe, right now I just put it into our QA asset scripts since I did it to investigate bugs and felt it had more value to keep than removing it, but maybe someone would be interested in it. I would say there is no obvious user-value in it, but maybe for ourselves. However, you likely want snapshots and this was simple and sufficient to solve such problems. I do not intend to move it there as part of this PR at least.

Reply:
However its handy to be able to see which InputActionReferences exist in memory and on disc as serialised SOs